Update pycmap_is_climatology.rst

MSchatzUW opened the pull request:
Megan's edits and comments

MSchatzUW committed:
Update pycmap_is_climatology.rst

Megan's edits and comments

norlandrhagen reviewed (commented):
mind removing comment

MSchatzUW committed:
Update pycmap_is_climatology.rst

Removed Comment:

Is there suppose to be a SQL example next? Not sure. Just checking.

MSchatzUW reviewed (commented):
Removed Comment:

Is there suppose to be a SQL example with this method?
